etl/steps/export/explorers/migration/latest/migration_flows.config.yml

```diff
@@ -0,0 +1,58 @@
+definitions:
+  common_views:
+    # Applies to all views with dimension metric=emigrants, unless overridden
+    - dimensions:
+        metric: emigrants
+    # Applies to all views with dimension metric=immigrants, unless overridden
+    - dimensions:
+        metric: immigrants
+
+config:
+  explorerTitle: Migration Flows
+  explorerSubtitle: Explore global migration data, country-by-country.
+  isPublished: true
+  selection:
+    - United States
+    - United Kingdom
+    - France
+    - Mexico
+    - India
+    - China
+    - Germany
+    - Australia
+    - Brazil
+    - South Africa
+    - World
+  thumbnail: https://ourworldindata.org/app/uploads/2022/03/migration-flows-explorer.png
+  subNavId:
+  subNavCurrentId:
+  hasMapTab: true
+  yAxisMin: 0
+  hideAnnotationFieldsInTitle: true
+  tab: map
+
+dimensions:
+  - name: Country
+    slug: country_select
+  - name: Metric
+    slug: metric
+    presentation:
+      type: radio
+    choices:
+      - slug: "immigrants"
```

Comment:
I would put Emigrants first as in the current explorer, just as the default view is Afghanistan and the 'Emigrants' slice is much more populated.